src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map(list => (
+        <div className="card" data-cy="Movie" key={list.imdbId}>
+          <MovieCard {...list} />
+        </div>
```

Comment:
```suggestion
          <MovieCard {...list} key={list.imdbId} />
```
move `div` into `MovieCard` component, and set `key` right for MovieCard